src/DataFixtures/UserFixtures.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace App\DataFixtures;
+
+use App\Entity\User;
+use Doctrine\Bundle\FixturesBundle\Fixture;
+use Doctrine\ORM\EntityManagerInterface;
+use Doctrine\Persistence\ObjectManager;
+use FOS\UserBundle\Model\UserManagerInterface;
+use FOS\UserBundle\Util\TokenGeneratorInterface;
+
+/**
+ * Creates a user to own packages.
+ */
+class UserFixtures extends Fixture
+{
+    public const PACKAGE_MAINTAINER_REFERENCE = 'package-maintainer';
+
+    private UserManagerInterface $userManager;
+
+    private TokenGeneratorInterface $tokenGenerator;
+
+    public function __construct(
+        UserManagerInterface $userManager,
+        TokenGeneratorInterface $tokenGenerator
+    ) {
+        $this->userManager = $userManager;
+        $this->tokenGenerator = $tokenGenerator;
+    }
+
+    public function load(ObjectManager $manager)
+    {
+        /** @var User $user */
+        $user = $this->userManager->createUser();
```

Comment:
I suggest using `new User()` here for simplicity. There is no reason to account for the customizable User class in project-level code.

Reply:
Fixed.